Benchmarks/Package.swift

```diff
@@ -0,0 +1,49 @@
+// swift-tools-version:5.9
+
+import PackageDescription
+
+let package = Package(
+    name: "benchmarks",
+    platforms: [
+        .macOS(.v13),
+    ],
+    dependencies: [
+        .package(path: "../"),
```

Comment:
I realize this is the way this tooling work, but it makes me shudder just the same 🤦‍♀️